src/styles.css

```diff
@@ -1,4 +1,6 @@
+/* AB Gilroy font family/style/weight source */
 @import 'https://unpkg.com/@abds/styles@3.3.0/dist/fonts.css';
+/* Apollo's implementation of reset css */
 @import 'https://unpkg.com/@abds/styles@3.3.0/dist/reset.css';
```

Comment:
Would it not be better to just pull the file from unpkg into the repo locally in case unpkg is not available for some reason?

Reply:
I can add a comment that they can do that as well to remove external dependencies but i just thought it is implied just like any other module

Reply:
Is it implied though? I'm not so sure I would count on that.

Reply:
added a comment in the readme to add clarity